geode-core/src/main/java/org/apache/geode/distributed/internal/locks/DLockRequestProcessor.java

```diff
@@ -196,6 +196,13 @@ long getLeaseExpireTime() {
     return this.response.leaseExpireTime;
   }
 
+  /**
+   *
+   * @param interruptible
+   * @param lockId
+   * @return
+   * @throws InterruptedException only possible if interruptible is true.
+   */
   protected boolean requestLock(boolean interruptible, int lockId) throws InterruptedException {
     final boolean isDebugEnabled_DLS = logger.isTraceEnabled(LogMarker.DLS);
```

Comment:
Did you want to remove this, as you seem to be removing a lot of switching based on it.  Is this code going to be log spammy without this?